### PR TITLE
docs: FIR-30099: update version in readme.md

### PR DESCRIPTION
**Pull request by alexradzin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ Installing the Package
 ======================
 
 Here is a FireboltNetSDK [NuGet page](https://www.nuget.org/packages/FireboltNetSDK/).
+Currently  installed version is 1.1.0. The preferable way to update version is using release GitHub action, however this can be done manually as well using CLI. 
+Here is how the manual installation can be done. 
 - Install using **.NET CLI**
 ```{r, engine='bash', code_block_name}
-dotnet add package FireboltNetSDK --version "(1.0.0-alpha,)"
+dotnet add package FireboltNetSDK --version 1.2.0
```
Comment (stepansergeevitch) [inline README.md:23]: remove the whole `--version 1.2.0`. Just let it install the latest version

File: README.md [hunk 2]
```diff
@@ -16,9 +16,11 @@ Installing the Package
 ======================
 
 Here is a FireboltNetSDK [NuGet page](https://www.nuget.org/packages/FireboltNetSDK/).
+Currently  installed version is 1.1.0. The preferable way to update version is using release GitHub action, however this can be done manually as well using CLI.
```
Comment (stepansergeevitch) [inline README.md:19]: I'm not sure what it's about. Users can't install the .net sdk using a github action. Also not sure what `Currently  installed version is 1.1.0` means, please explain

File: README.md (hunk 1 above)
Comment (alexradzin) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (alexradzin) [reply README.md]: removed

**Review by stepansergeevitch — APPROVED** [review]

**Pull request merged**